src/molecules/tabBar/component.tsx

```diff
@@ -14,13 +16,19 @@ export interface ITabBarTheme {
 
 interface IStyledTabBarProps {
   theme: ITabBarTheme;
+  contentAlignment: ResponsiveField<Alignment>;
 }
 
+const getContentAlignmentCss: CssConverter<Alignment> = (field: Alignment): string => {
+  return `justify-content: ${getFlexContentAlignment(field)};`;
+};
+
 const StyledTabBar = styled.div<IStyledTabBarProps>`
   display: flex;
   flex-direction: row;
   max-width: 100%;
   overflow: auto;
+  ${(props: IStyledTabBarProps): string => fieldToResponsiveCss(props.contentAlignment, useDimensions(), getContentAlignmentCss)};
```

Comment:
The problem is with this. I think so
Please see the `stack` component, it's implemented in the same way